applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -117,6 +117,10 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 		}
 	}
 
+#if (CONFIG_BT_LL_ACS_NRF53)
```

Comment:
If ``ble_hci_vs.h`` is included we should use ``if (IS_ENABLED(CONFIG_BT_LL_ACS_NRF53))`` here

Reply:
It is not. We will handle this seperately